cloudify/context.py

```diff
@@ -352,6 +352,11 @@ def id(self):
         """The node instance id."""
         return self._context.get('node_id')
 
+    @property
```

Comment:
Any reason not to just make self._node be self.node?

Reply:
consistency with all the other ones

Reply:
and i guess self._node is a restclient node object, only used internally in this, and this is a NodeContext